vnc/vnc.c

```diff
@@ -255,7 +305,364 @@ lib_process_channel_data(struct vnc *v, int chanid, int flags, int size,
     return 0;
 }
 
-/******************************************************************************/
+/**************************************************************************//**
```

Comment:
Is the last `//**` intended?

Reply:
It's a doxygen-compatible comment - see paragraph 4 of [https://www.doxygen.nl/manual/docblocks.html](https://www.doxygen.nl/manual/docblocks.html).

Reply:
Understood. Doxygen is a nice tool to generate documents. IMO it's okay to mix the original style and Doxygen style.